.devcontainer/devcontainer.json

```diff
@@ -39,7 +39,7 @@
 		"rust-analyzer.rustfmt.enableRangeFormatting": true,
 		"[rust]": {
 			"editor.defaultFormatter": "rust-lang.rust-analyzer",
-			"editor.formatOnSave": true
+			"editor.formatOnSave": false
```

Comment:
not sure I aggree. are you 100% that extensions fromatter is 100% same as CI one?